Variant.mpl

```diff
@@ -118,6 +121,8 @@ Variant: [{
     @memory storageAddress index @typeList @ addressToReference
```

Comment:
Since we introduced `getUnchecked` function, we should replace line 118 by next one:
```
index getUnchecked
```